web/router.ex

```diff
@@ -9,12 +10,23 @@ defmodule Community.Router do
     plug :put_secure_browser_headers
   end
 
+  pipeline :require_admin do
+    plug Community.RequireAdmin
+  end
+
   scope "/", Community do
     pipe_through :browser
 
     get "/", HomePageController, :show, as: :root
     get "/pages/:id", PageController, :show
     resources "/jobs", JobController
     resources "/members", MemberController, only: [:index, :new, :create]
+
+    resources "/session", SessionController, only: [:new, :create]
+  end
+
+  scope "/admin", ExAdmin do
+    pipe_through [:browser, :require_admin]
```

Comment:
You can call plugs directly in pipelines. So this could be `pipe_through [:browser, Community.RequireAdmin]`